server/model/user.model.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-console */
+import { Pool } from "pg";
+
+const pool = new Pool({
+	user: process.env.DB_USER,
+	host: process.env.DB_HOST,
+	database: process.env.DB_NAME,
+	password: process.env.DB_PASS,
+	port: process.env.DB_PORT,
+});
+
```

Comment:
Hey Fatemeh. Could you please remove 

import { Pool } from "pg";

const pool = new Pool({
	user: process.env.DB_USER,
	host: process.env.DB_HOST,
	database: process.env.DB_NAME,
	password: process.env.DB_PASS,
	port: process.env.DB_PORT,
});

And add Import db from "/db";